profile/README.md

```diff
@@ -10,3 +10,44 @@
 🍿 Fun facts - what does your team eat for breakfast?
 🧙 Remember, you can do mighty things with the power of [Markdown](https://docs.github.com/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)
 -->
+
+As “Big Data” researchers, we both apply existing techniques and continuously work to improve them and create new methods. Much of our work to date has focused on prostate cancer, but we are increasingly applying the same techniques to cancers of the kidney, bladder and other sites, and discovering fascinating commonalities across them.
```

Comment:
change to "of the kidney, bone and connective tissue (sarcomas), brain, thyroid, ovary and other sites, and..."